lua/lspconfig/configs/stylua3p_ls.lua

```diff
@@ -0,0 +1,16 @@
+local util = require('lspconfig.util')
+
+return {
+  default_config = {
+    cmd = { 'stylua-3p-language-server' },
+    filetypes = { 'lua' },
```

Comment:
stylua also supports `luau`: https://luau.org/

Reply:
the LS does not currently support it